### DISCUSSION TITLE
Catalog Consumer app

### DISCUSSION THREAD

**Discussion by miguelverweij** [microsoft/PowerPlatform-Catalog#18] [Ideas]
AFAIK it is currently only possible to get the artifacts out of the Catalog by using the PAC CLI. I think the solution as a whole could benefit from a tailored Catalog Consumer application. I think Citizen Developers will benefit tremendously from the functionality it has to offer, but using a CLI isn't really Citizen Dev friendly in my opinion. If we could have an interface where Catalog Consumers could browse through the artifacts and deploy them to the environment they work in (potentially even spinning up a dev environment) would make it much more acessible.

**Comment by MattB-msft** [comment]
There is a private preview getting started on just this subject shortly. 
We will have an integration with the various Make.* for submission and consumption flows.  If this is something you're interested in participating in, we can look to get you added. 

@derekkwanpm

**Comment by derekkwanpm** [comment]
hi @miguelverweij, can you email me at derekkwan@microsoft.com with your tenant ID? i can also provide you more info over email. thank you!

**Comment by miguelverweij** [comment]
I would love to join the private preview. I just emailed you @derekkwanpm !
